api/apiFunctions.ts

```diff
@@ -60,6 +61,30 @@ export async function logoutAccount(): Promise<{}> {
   }
 }
 
+/**
+ * Get the current user
+ *
+ * @return {Models.DocumentList<Models.Document> | Error} - The user object or an error
+ */
+export async function getCurrentUser(userId: IUser['id']): Promise<IUser> {
```

Comment:
Why was this added into this PR? Do we not already get the user?

Reply:
Yes we get the User which returns appwrites AUTH USER table.  We need to then fetch the User table we created that has the leagues they are a part of which is not in the APPWRITE AUTH TABLE.

Reply:
This is fetching our own custom user table correct?

Reply:
@shashilo  that is correct!